java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java

```diff
@@ -380,15 +380,16 @@ private static String createExtnPattern(String singleExtnSymbols) {
 
   /**
    * INTERNATIONAL and NATIONAL formats are consistent with the definition in ITU-T Recommendation
-   * E123. For example, the number of the Google Switzerland office will be written as
-   * "+41 44 668 1800" in INTERNATIONAL format, and as "044 668 1800" in NATIONAL format.
-   * E164 format is as per INTERNATIONAL format but with no formatting applied, e.g.
-   * "+41446681800". RFC3966 is as per INTERNATIONAL format, but with all spaces and other
-   * separating symbols replaced with a hyphen, and with any phone number extension appended with
-   * ";ext=". It also will have a prefix of "tel:" added, e.g. "tel:+41-44-668-1800".
+   * E.123. However we follow local conventions such as using '-' instead of whitespace as
+   * separators. For example, the number of the Google Switzerland office will be written as
+   * "+41 44 668 1800" in INTERNATIONAL format, and as "044 668 1800" in NATIONAL format. E164
+   * format is as per INTERNATIONAL format but with no formatting applied, e.g. "+41446681800".
+   * RFC3966 is as per INTERNATIONAL format, but with all spaces and other separating symbols
+   * replaced with a hyphen, and with any phone number extension appended with ";ext=". It also
+   * will have a prefix of "tel:" added, e.g. "tel:+41-44-668-1800".
    *
    * Note: If you are considering storing the number in a neutral format, you are highly advised to
-   * use the PhoneNumber class.
+   * use the phonenumber.proto. For details, see http://go/phone-pb
```

Comment:
Please revert the change on this line. Github uses a Java class, not a proto.

Reply:
Oh Sorry. My bad. Reverted this change now. PTAL.